integ-tests/src/cucumber/java/steps/dataservice/DescribeEnvironmentSteps.java

```diff
@@ -0,0 +1,203 @@
+public class DescribeEnvironmentSteps implements En {
+
+  @Autowired private DataServiceWrapper dataServiceWrapper;
+  private static final String ACCOUNT_ID = "accountID";
+  private static final String TASK_DEFINITION_ARN =
+      "arn:aws:ecs:us-east-1:" + ACCOUNT_ID + ":task-definition/sleep";
+  private static final String ROLE_ARN = "arn:aws:iam::" + ACCOUNT_ID + ":role/testRole";
+  private static final String CLUSTER_NAME_PREFIX = "cluster";
+  private String postfix = null;
+
+  public DescribeEnvironmentSteps() {
+    Given(
+        "^I create an environment named \"([^\"]*)\"$",
+        (final String environmentNamePrefix) -> {
+          this.postfix = generatePostfix();
+          dataServiceWrapper.createEnvironment(createEnvironmentRequest(environmentNamePrefix));
+        });
+
+    When(
+        "^I describe the environment named \"([^\"]*)\"$",
+        (final String environmentNamePrefix) -> {
+          dataServiceWrapper.describeEnvironment(describeEnvironmentRequest(environmentNamePrefix));
+        });
+
+    Then(
+        "^the created and described environments match$",
+        () -> {
+          final CreateEnvironmentResponse createEnvironmentResponse =
+              dataServiceWrapper.getLastFromHistory(CreateEnvironmentResponse.class);
+          final DescribeEnvironmentResponse describeEnvironmentResponse =
+              dataServiceWrapper.getLastFromHistory(DescribeEnvironmentResponse.class);
+          checkEnvironmentEquality(
+              createEnvironmentResponse.getEnvironment(),
+              describeEnvironmentResponse.getEnvironment());
+        });
+
+    Given(
+        "^I update the environment named \"([^\"]*)\"$",
+        (final String environmentNamePrefix) -> {
+          dataServiceWrapper.updateEnvironment(updateEnvironmentRequest(environmentNamePrefix));
+        });
+
+    Then(
+        "^the updated and described environments match$",
+        () -> {
+          final UpdateEnvironmentResponse updateEnvironmentResponse =
+              dataServiceWrapper.getLastFromHistory(UpdateEnvironmentResponse.class);
+          final DescribeEnvironmentResponse describeEnvironmentResponse =
+              dataServiceWrapper.getLastFromHistory(DescribeEnvironmentResponse.class);
+          checkEnvironmentEquality(
+              updateEnvironmentResponse.getEnvironment(),
+              describeEnvironmentResponse.getEnvironment());
+        });
+
+    When(
+        "^I try to describe a non-existent environment named \"([^\"]*)\"$",
+        (final String environmentNamePrefix) -> {
+          dataServiceWrapper.tryDescribeEnvironment(
+              describeEnvironmentRequest(environmentNamePrefix));
+        });
+
+    Given(
+        "^I delete the created environment named \"([^\"]*)\"$",
+        (final String environmentNamePrefix) -> {
+          dataServiceWrapper.tryDeleteEnvironment(deleteEnvironmentRequest(environmentNamePrefix));
```

Comment:
if the step is not described with "try" you shouldn't call the wrapper try methods since they will expect the call to fail.

Reply:
Sure!